lib/plugins/aws/deploy/compile/events/schedule/index.js

```diff
@@ -38,9 +40,19 @@ class AwsCompileScheduledEvents {
               }
               ScheduleExpression = event.schedule.rate;
               State = event.schedule.enabled ? 'ENABLED' : 'DISABLED';
+              Input = event.schedule.input || '';
+              InputPath = event.schedule.inputPath || '';
```

Comment:
@lmammino great catch! IMO we should throw an error on compilation before deployment if both are set. This is better UX as the feedback loop is much faster and the user doesn't have to wait to upload his service and update the stack to hit the error.